com.unity.template-hd/Packages/com.unity.template.hd/package.json

```diff
@@ -7,9 +7,16 @@
   "unity": "2022.1",
   "description": "This template utilizes the High Definition Render Pipeline. Making it a good starting point for people focused on high-end graphics that want to develop games for platforms that support Shader Model 5.0 (DX11 and above). \n\nBeyond being tuned for High End visuals this project includes\n\u25aa Shadergraph \n\u25aa Visual Effect Graph  \n\u25aa Presets \n\u25aa Example content",
   "dependencies": {
+    "com.unity.collab-proxy": "1.9.0",
+    "com.unity.feature.development": "1.0.1",
+    "com.unity.textmeshpro": "3.0.6",
     "com.unity.cinemachine": "2.8.0",
+    "com.unity.feature.development": "1.0.1",
     "com.unity.inputsystem": "1.1.0-pre.5",
     "com.unity.learn.iet-framework": "2.0.0",
-    "com.unity.render-pipelines.high-definition": "13.0.0"
+    "com.unity.learn.iet-framework.authoring": "1.0.0",
```

Comment:
I have no idea. Guess it is require to work with IET no? so any change to iet will be required. Guess we should let him otherwise it will be painful to work each time dev wants to do a change in iet? 
cc @iM0ve  @pierre-unity

Reply:
You can keep this package in manifest.json only, this way internal developers can still make changes to IET without problems. Including it in package.json will include it in the final template, so end users will have access to it and will be able to make changes to the tutorial. The authoring package does not need to be included in package.json if the intention is only to present tutorials to end users.

Reply:
I see, ok